client/src/app/tabs/rpa/RPAEditor.js

```diff
@@ -134,6 +145,17 @@ export class RPAEditor extends CachedComponent {
     return editor;
   }
 
+  handleError(error) {
+    this.setCached({
+      editor: null,
+      lastXML: null,
+    });
+
+    console.log(error);
```

Comment:
Is this our standard pattern? I'd expect that we'd log with a message, or `console.error(...)`, or not log here at all (because it will be handled centrally).

The BPMN editor does not log anything ([ref](https://github.com/camunda/camunda-modeler/blob/develop/client/src/app/tabs/cloud-bpmn/BpmnEditor.js#L324)).

Reply:
No, it is not a pattern. Just something I missed to remove from my debugging session 🙈